### PR TITLE
Added link to v4.0.0 google_bigquery documentation

### PR DESCRIPTION
**Pull request by robbavey**
The tagged version of v4.0.0 google big query plugin contains
error which causes the build to fail. Include this correct version
to fix the build

### CODE REVIEW

File: docs/versioned-plugins/outputs/google_bigquery-v4.0.0.asciidoc [hunk 1]
```diff
@@ -0,0 +1,349 @@
+:plugin: google_bigquery
+:type: output
+:default_codec: plain
+
+///////////////////////////////////////////
+START - GENERATED VARIABLES, DO NOT EDIT!
+///////////////////////////////////////////
+:version: %VERSION%
```
Comment (jsvd) [inline docs/versioned-plugins/outputs/google_bigquery-v4.0.0.asciidoc]: one detail I forgot to mention, we need to fill out these variables. You can see an example here: https://raw.githubusercontent.com/elastic/logstash-docs/versioned_plugin_docs/docs/versioned-plugins/outputs/google_bigquery-v3.2.1.asciidoc

**Comment by dedemorton** [comment]
The script for generating the versioned doc also does some cleanup on the links. Rather than hacking this manually, I'd suggest running the script to generate the docs. Let me know if you want me to do that.

**Comment by robbavey** [comment]
@dedemorton Thanks - can you point me to some instructions? I'd like to be able to do it myself in the event that this needs to be done again...

**Comment by dedemorton** [comment]
@robbavey I had a PR open that contained the step, but removed them because we have a CI job now. I'll send you the steps through email. The other option is to look at the script and make the manual hacks:

https://github.com/elastic/docs-tools/blob/master/versioned_plugins.rb#L200

**Comment by robbavey** [comment]
Thank you @karenzone!

**Pull request merged**